Tools/TestWebKitAPI/Tests/WebKit/red.html

```diff
@@ -0,0 +1 @@
+<body style="background-color: red"></body>
```

Comment:
Could just be inline in the test, but this is fine too

Reply:
i tried doing that but it didnt seem to work correctly with `goBack`